migrations/5.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pokestop
+ADD COLUMN `ar_scan_eligible` tinyint(1) unsigned DEFAULT 0;
```

Comment:
This is what I was talking about. Ideally we want migrations to be generated automatically: https://www.npmjs.com/package/sequelize-auto-migrations

Reply:
I personally prefer the manual migration creations but that's just me.